Hercules-DJControl-Inpulse-500-script.js

```diff
@@ -968,15 +1042,14 @@ DJCi500.init = function() {
   engine.makeConnection("[Channel4]", "play_indicator", DJCi500.numberIndicator);
   engine.getValue("[Channel4]", "play_indicator", DJCi500.numberIndicator);
 
-  // Connect Master VU meter 
-  engine.makeConnection("[Master]", "VuMeterL", DJCi500.vuMeterUpdateMaster);
-  engine.makeConnection("[Master]", "VuMeterR", DJCi500.vuMeterUpdateMaster);
-  engine.makeConnection("[Master]", "PeakIndicatorL", DJCi500.vuMeterPeakLeftMaster);
-  engine.makeConnection("[Master]", "PeakIndicatorR", DJCi500.vuMeterPeakRightMaster);
+  // Connect Master VU meter
+  engine.makeConnection("[Main]", "vu_meter_left", DJCi500.vuMeterUpdateMaster);
+  engine.makeConnection("[Main]", "vu_meter_right", DJCi500.vuMeterUpdateMaster);
+  engine.makeConnection("[Main]", "peak_indicator_left", DJCi500.vuMeterPeakLeftMaster);
+  engine.makeConnection("[Main]", "peak_indicator_right", DJCi500.vuMeterPeakRightMaster);
 
-  engine.getValue("[Master]", "VuMeterL", DJCi500.vuMeterUpdateMaster);
-  engine.getValue("[Master]", "VuMeterR", DJCi500.vuMeterUpdateMaster);
-  engine.getValue("[Controls]", "AutoHotcueColors", "DJCi500.AutoHotcueColors");

```

Comment:
If this is removed, maybe its also good to remove: https://github.com/resetreboot/herimp-500map/pull/13/files#diff-79ef78dd63564d6a922d1bb975c4726c8139712a872479a14dafb7460fb0d479R1006 ?

Reply:
I'm sorry, but your link doesn't highlight what you think should be removed.

Reply:
This now works ? https://github.com/resetreboot/herimp-500map/blob/e06d2a5157cfc496cd5e784b9b937f0ae50c7b3b/Hercules-DJControl-Inpulse-500-script.js#L1009

If here you are removing:

```
engine.getValue("[Controls]", "AutoHotcueColors", "DJCi500.AutoHotcueColors");
```

This seems not to be needed anymore:

```
 DJCi500.AutoHotcueColors = true;
 ```

Reply:
Good catch, now Github even included the line in your comment.